js/meetup/data.js

```diff
@@ -1,4 +1,43 @@
 const meetups = [
+  {
+    episode: 17,
```

Comment:
This should be 19 as we have already done 2 remote meetups in may and june
https://www.meetup.com/Nepal-Ruby-Users-Group/events/past/

Reply:
Updated to 19